examples/animation/double_pendulum.py

```diff
@@ -64,8 +63,13 @@ def derivs(state, t):
 # initial state
 state = np.radians([th1, w1, th2, w2])
 
-# integrate your ODE using scipy.integrate.
-y = integrate.odeint(derivs, state, t)
+# integrate the ODE using Euler's method
+y = np.empty((len(t), 4))
+y[0] = state
+for i in range(1, len(t)):
+    y[i] = y[i - 1] + derivs(t[i - 1], y[i - 1]) * dt
+# A more accurate estimate could be obtained e.g. using scipy
+# (`y = scipy.integrate.solve_ivp(derivs, t[[0, -1]], state, t_eval=t).y.T`).
```

Comment:
Let’s leave out the markup. This is a code comment and will not be styled.

```suggestion
#     y = scipy.integrate.solve_ivp(derivs, t[[0, -1]], state, t_eval=t).y.T
```

Reply:
sure